tests/assistant_spec.js

```diff
@@ -553,6 +565,43 @@ describe('TranslationAssistant (math-translate)', function() {
         assertSuggestions(allItems, itemsToTranslate, translatedStrs, 'de');
     });
 
+    it('should translate math notations based on template', function() {
+        const allItems = [{
+            englishStr: 'simplify $3\\times x = 9.9 \\div 3$',
+            translatedStr: 'simplifyz $3\\cdot x = 9{,}9 \\mathbin{:} 3$',
+        }];
+        const itemsToTranslate = [
+            {englishStr: 'simplify $5\\times x=0.9\\div 3$', translatedStr: ''},
+        ];
+        const translatedStrs = ['simplifyz $5\\cdot x=0{,}9\\mathbin{:} 3$'];
+        assertSuggestions(allItems, itemsToTranslate, translatedStrs, 'lol');
+    });
```

Comment:
I think it would not pass because `lol` lang is in `MAYBE_TIMES_AS_CDOT` and not in `TIMES_AS_CDOT`. But you're right this test is not very instructive, I've modified it and added more tests that will hopefully make that clearer.